components/match2/commons/match_group_input.lua

```diff
@@ -312,10 +312,13 @@ end
 
 --[[
 Merges an opponent struct into a match2 opponent record.
+
+If any property exists in both the record and opponent struct, the value from the opponent struct will be prioritized.
+Reason for this is that the opponent struct is retrived programmatically via Module:Opponent.
```

Comment:
```suggestion
Reason for this is that the opponent struct is retrieved programmatically via Module:Opponent.
```

And answer the following question in the comment:

Which gives us what advantage?